R/import_data.R

```diff
@@ -277,7 +276,7 @@ iNZread.dta <- function(obj, ...) {
                                        convert.dates   = TRUE,
                                        convert.factors = TRUE)
   
-  attr(temp.data.frame, "full.file") = TRUE
+  attr(temp.data.frame, "preview") = obj$preview
```

Comment:
This (and L301) can be removed?

If the function is incapable of reading only part of a file, then it always returns the full file, as expected.